pages/index.js

```diff
@@ -32,19 +31,24 @@ export default function Home({ devices }) {
           <option value="Lighting">Lighting</option>
         </select>
         <button type="submit">Search</button>
+        <button type="button" onClick={() => setIsFiltered(true)}>
+          Reset
+        </button>
       </form>
       {isFiltered ? (
         <StyledList>
           {devices.map((device) => (
             <Wrapper key={device.id}>
               <Card
```

Comment:
You can also create a component at this part.
It's up to you. If you don't need those Card components elsewhere it doesn't matter.